### ISSUE TITLE
Error processing vulnerabity alerts

### ISSUE THREAD

**Issue by gorohash** [renovatebot/renovate#11911]
### How are you running Renovate?

Self-hosted

### Please select which platform you are using if self-hosting.

GitHub Enterprise Server

### If you're self-hosting Renovate, tell us what version of Renovate you run.

27.19.2

### Describe the bug

The following error messages are printed during the execution of renovate.

### Relevant debug logs

<details><summary>Logs</summary>

```
ERROR: Error processing vulnerabity alerts (repository=gorohash/renovate-playground)
       "err": {
         "message": "Cannot read properties of null (reading 'package')",
         "stack": "TypeError: Cannot read properties of null (reading 'package')\n    at Proxy.getVulnerabilityAlerts (/home/circleci/.npm/_npx/05eeecd92f4e18e0/node_modules/renovate/dist/platform/github/index.js:1298:108)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async detectVulnerabilityAlerts (/home/circleci/.npm/_npx/05eeecd92f4e18e0/node_modules/renovate/dist/workers/repository/init/vulnerability.js:48:20)\n    at async initRepo (/home/circleci/.npm/_npx/05eeecd92f4e18e0/node_modules/renovate/dist/workers/repository/init/index.js:31:14)\n    at async Object.renovateRepository (/home/circleci/.npm/_npx/05eeecd92f4e18e0/node_modules/renovate/dist/workers/repository/index.js:64:18)\n    at async Object.start (/home/circleci/.npm/_npx/05eeecd92f4e18e0/node_modules/renovate/dist/workers/global/index.js:116:13)\n    at async /home/circleci/.npm/_npx/05eeecd92f4e18e0/node_modules/renovate/dist/renovate.js:33:24"
       }
 WARN: Error parsing vulnerability alert (repository=gorohash/renovate-playground)
       "err": {
         "message": "Cannot read properties of null (reading 'firstPatchedVersion')",
         "stack": "TypeError: Cannot read properties of null (reading 'firstPatchedVersion')\n    at detectVulnerabilityAlerts (/home/circleci/.npm/_npx/05eeecd92f4e18e0/node_modules/renovate/dist/workers/repository/init/vulnerability.js:75:46)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async initRepo (/home/circleci/.npm/_npx/05eeecd92f4e18e0/node_modules/renovate/dist/workers/repository/init/index.js:31:14)\n    at async Object.renovateRepository (/home/circleci/.npm/_npx/05eeecd92f4e18e0/node_modules/renovate/dist/workers/repository/index.js:64:18)\n    at async Object.start (/home/circleci/.npm/_npx/05eeecd92f4e18e0/node_modules/renovate/dist/workers/global/index.js:116:13)\n    at async /home/circleci/.npm/_npx/05eeecd92f4e18e0/node_modules/renovate/dist/renovate.js:33:24"
       }
```

</details>

### Have you created a minimal reproduction repository?

No reproduction repository

**Comment by gorohash** [comment]
`securityVulnerability` field of `RepositoryVulnerabilityAlert` is nullable.
https://docs.github.com/en/graphql/reference/objects#repositoryvulnerabilityalert

However, it is not null checked in following code.
https://github.com/renovatebot/renovate/blob/3ca6855d4c687fa38cc50b598c2d1f04436c1efe/lib/platform/github/index.ts#L1597

It seems to be the cause.

**Comment by aleixhub** [comment]
I couldn't reproduce it in github.com, only using the github enterprise v2.22.7

**Comment by aleixhub** [comment]
> So is there no "fixed version" in github 2.22.7? 
In v2.22.7 I never got any value.. always I get the same result -> `"firstPatchedVersion": null,`

> Would it need to be a hack like `vulnerableVersionRange.replace('< ', '')`?
Yes, that could be one possible solution. Taking into account that there are other kinds of annotations like ` "vulnerableVersionRange": ">= 2.0.0, < 2.17.2"`
